IEDiagnosticsAdapter/IEDiagnosticsAdapter.cpp

```diff
@@ -354,7 +354,7 @@ void IEDiagnosticsAdapter::OnHttp(websocketpp::connection_hdl hdl)
             url.Replace('\\', '/');
             url.Replace(" ", "%20");
             url.Replace("file://", "file:///");
-            CStringA title(it.second.title);
+            CStringA title = Helpers::EscapeJsonString(it.second.title);
             CStringA fileName(::PathFindFileNameW(it.second.filePath));
```

Comment:
For sure, it needs to be escaped.

![13d9dab5-ea85-4b8a-be86-6d28bff94070](https://cloud.githubusercontent.com/assets/9012/6955669/6b20db1c-d91c-11e4-8fae-37d34f6cce01.png)